### PR TITLE
TH3 test parameters for Dell Z9332 systems

### PR DESCRIPTION
**Pull request by vmittal-msft**
### Description of PR

Adding test infrastructure parameters for TH3 based systems

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Adding test infrastructure parameters for TH3 based system

#### How did you do it?
Updated QoS related test files

#### How did you verify/test it?
verified on T0 topology of Dell Z9332 systems.

#### Supported testbed topology if it's a new test case?
T0

### CODE REVIEW

File: tests/qos/files/qos.yml [hunk 1]
```diff
@@ -1309,3 +1309,155 @@ qos_params:
                 cell_size: 208
             hdrm_pool_wm_multiplier: 4
             cell_size: 208
+    th3:
```
Comment (gechiang) [inline tests/qos/files/qos.yml:1312]: How doe the variables "broadcom_th3_hwskus" gets interpreted as "th3" here?  is there something already handling that?

Comment (vmittal-msft) [reply tests/qos/files/qos.yml:1312]: Yes there is an exiting logic which handles it for all ASIC types the same way.

**Review by neethajohn — APPROVED** [review]

File: tests/qos/files/qos.yml [hunk 2]
```diff
@@ -1309,3 +1309,155 @@ qos_params:
...
+        topo-any:
```
Comment (119064273) [inline tests/qos/files/qos.yml:1313]: hi vmittal:
     we are testing qos sai base on your change, but we meet some questions, can you give some advice?
     1: how you get TH3 qos.yml datas?
     2: where to get ptf-brcm?

**Review by 119064273 — COMMENTED** [review]
hi  @vmittal-msft , can you share how get qos.yml datas ?

**Pull request merged**